homeassistant/components/reolink/entity.py

```diff
@@ -178,8 +178,13 @@ def __init__(
             else:
                 self._dev_id = f"{self._host.unique_id}_ch{dev_ch}"
 
+            connections = set()
+            if (mac := self._host.api.baichuan.mac_address(dev_ch)) != "Unknown":
```

Comment:
Can we move this check to the lib as it has protocol-specific details?
I would like more a function, which will return the mac address or None (if not available)

Reply:
I shot myself a bit in the foot here:
https://github.com/starkillerOG/reolink_aio/blob/64b92011288f205a1fc981e6c504dc3c6ea9f3e7/reolink_aio/baichuan/baichuan.py#L1447C1-L1448C73

By default it would return None, but I made it such that it would always be a string ("Unknown" if None) because I thought that would be more conveniant (like I do with model).

But in this case not so much.
Will change it in the upstream, but will need to wait for another version bump.

Reply:
Done